### DISCUSSION TITLE
can't instal

### DISCUSSION THREAD

**Discussion by ArthaMaulana** [nigification/Flowgorithm-Mac-OSX#24] [Q&A]
<img width="786" alt="image" src="https://github.com/jostasik/Flowgorithm-macOS/assets/145268291/cff14108-bbd7-4c11-aa2a-e922abd83069">

can you tell me, what's wrong? 
my device is MBA M1 2020

sorry my English is bad

**Comment by jostasik** [comment]
Hello sorry i think i had an error in one of my files.. I just changed it so try running these again:

brew tap jostasik/homebrew-tap
brew update
brew install jostasik-flowgorithm

**Reply by ArthaMaulana** [reply]
I  succeed install the app but I can open my file in the app, can you tell me why?
thank you for answering

**Reply by jostasik** [reply]
Okay with Flowgorithm running, try dragging the file directly into the Flowgorithm window!

**Reply by ArthaMaulana** [reply]
THANKS BRO, ITS WORKING

**Reply by ArthaMaulana** [reply]
umm bro, how to share my file? i can't find my file in finder
thanks for answering

**Reply by jostasik** [reply]
@ArthaMaulana When you need to save something follow these steps! This is where your normal Mac user folders are. 😊 
![](https://github.com/jostasik/Flowgorithm-macOS/assets/31175268/2a1bae50-dffa-4934-9c82-be50bcbf6461)
